socs/agents/ucsc_radiometer/agent.py

```diff
@@ -68,7 +68,11 @@ def acq(self, session, params=None):
         session.set_status('running')
 
         while self.take_data:
-            r = requests.get(self.url)
+            try:
+                r = requests.get(self.url)
+            except ValueError:
```

Comment:
The error looks like it's raising `requests.exceptions.JSONDecodeError`, not a `ValueError`. I think we should be catching that instead here.

Reply:
(I did check that JSONDecodeError is a subclass of ValueError -- so this would catch it.)

Reply:
Ah, that makes sense.